src/Component/BookingTable/index.js

```diff
@@ -31,37 +66,40 @@ function BookingTable() {
                         <Typography variant="h6" color={'primary'}>First Name</Typography>
                         <TextField sx={{ color: 'primary' }}
                             required
-                            id="filled-required"
+                            id="filled-requireda"
                             label="Required"
                             variant="filled"
                         />
                         <Typography variant="h6" color={'primary'}>Last Name</Typography>
                         <TextField
                             required
-                            id="filled-required"
+                            id="filled-requiredb"
                             label="Required"
                             variant="filled"
                         />
                         <Typography variant="h6" color={'primary'}>Email Adress</Typography>
                         <TextField
                             required
-                            id="filled-required"
+                            id="filled-requiredc"
                             label="Required"
                             variant="filled"
                         />
                         <Typography variant="h6" color={'primary'}>Phone Number</Typography>
                         <TextField
                             required
-                            id="filled-required"
+                            id="filled-requiredd"
                             label="Required"
                             variant="filled"
                         />
                     </Box>
                 </Grid>
 
             </Grid>
-            <Box sx={{ marginLeft: 40, marginTop: 5 }}> <MainButton text={'Continue to Book'} /></Box>
-            <Box sx={{ marginLeft: 40, marginTop: 2, marginBottom: 3 }}> <Button sx={{ backgroundColor: '#F5F6F8', color: "#B3B3B3", width: 200 }}>Cancel</Button></Box>
+            <Box sx={{ marginLeft: 40, marginTop: 5 }}> <Link onClick={validate} style={{ textDecoration: "none" }} to={`/Payment/${productId}`}>
```

Comment:
we need to use a button instead of <Link> so we navigate to the next payment step if validation return true
![image](https://github.com/MohammadGhawadrah/Stayction/assets/132939680/ebc79fc7-409b-4f54-92ee-2baa14e54e36)